pkg/service/service.go

```diff
@@ -186,10 +186,15 @@ func List(client *occlient.Client, applicationName string) ([]ServiceInfo, error
 	// Iterate through serviceInstanceList and add to service
 	for _, elem := range serviceInstanceList {
 		conditions := elem.Status.Conditions
+		var status string
 		if len(conditions) == 0 {
-			return nil, fmt.Errorf("no condition in status for %+v", elem)
+			glog.Warningf("no condition in status for %+v, marking it as Unknown", elem)
```

Comment:
I know I was away for 3 weeks on PTO, but have we started using glog.Warningf now instead of the debug / erroring out?

Reply:
I wanted to be able to surface the fact that there is something weird going on with the status without returning an error (which was actually causing the issue Jeff was seeing). I'm all hears for a better option if you have one.